test/rails/action_controller/test_expectations.rb

```diff
@@ -6,6 +6,7 @@ class TestActionControllerExpectations < ActionController::TestCase
   def test_must_respond_with
     get :index
     must_respond_with :success
```

Comment:
Both ways are supported. Technically `must_respond_with` is an alias of `assert_response`, not an expectation. I suppose those aliases may be deprecated in the future, but that isn't a change I plan on making ATM.